pgrx-pg-config/src/lib.rs

```diff
@@ -491,17 +491,14 @@ impl<'a> PgConfigSelector<'a> {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Error)]
 pub enum PgrxHomeError {
+    #[error("You don't seem to have a home directory")]
     NoHomeDirectory,
+    #[error("$PGRX_HOME does not exist")]
```

Comment:
note that the `PathBuf` here is not printed. Should it?

Reply:
Hm, no. We want to be careful about printing paths or otherwise enumerating the environment in a way that could automatically pass-through to IO, because this crate doesn't know what the security context is for this error being formatted.

Reply:
thx, i will add a comment about that